inspirehep/orcid/ext.py

```diff
@@ -23,6 +25,7 @@ def __init__(self, app=None):
     def init_app(self, app):
         self.init_config(app)
         self.configure_appmetrics(app)
+        app.cli.add_command(import_legacy_tokens)
```

Comment:
`orcid is the name of the group`
```suggestion
        app.cli.add_command(orcid)
```

Reply:
ops